src/test/java/guitests/PanelFocusTest.java

```diff
@@ -148,6 +146,8 @@ private void createDefaultPanels() {
         panels.add(test3);
 
         globalConfig.setPanelInfo(panels);
-        configFileHandler.saveGlobalConfig(globalConfig);
+        ConfigFile globalConfigFile =
```

Comment:
Can we change all the variable names in tests from `globalConfigFile` to `sessionConfigFile`? For consistency.